src/app/services/project_service.js

```diff
@@ -333,6 +333,87 @@ angular
         return objects
     }
 
+     function assignSearchRelevance(results, q) {
```

Comment:
Would it be easier now to have unit tests, if the function is independent from jquery and the DOM?